doc/usersguide/UnderstandingRayHistoryState.rst

```diff
@@ -1,17 +1,17 @@
-Understanding ray history state in DAG-MCNP5
-============================================
+Understanding ray history state in DAG-MCNP5/6
+==============================================
 
-Documenting calls to the track() method in MCNP5
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Documenting calls to the track() method in MCNP
```

Comment:
Or maybe at some point (near the beginning) make it very clear that DAG-MCNP is referring to behavior for DAG-MCNP5/6 and leave it at that?